<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>projects | Giancarlo Croce</title> <meta name="author" content="Giancarlo Croce"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://giancarlocroce.github.io/projects/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Giancarlo </span>Croce</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/"></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">Curriculum vitae</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">projects</h1> <p class="post-description"></p> </header> <article> <h2 id="t-cell---epitope-interactions">T cell - epitope interactions</h2> <p>T cells play a vital role in our body’s defence against viruses, bacteria and even cancer. T cells can recognize molecular fragments presented by infected or cancerous cells (the so-called epitopes) and initiate a response to eliminate them. Below a video of a <strong>T cell targeting and killing a cancer cell</strong>.</p> <p><img align="left" src="../assets/img/t_cell_killing_cancer.gif" width="300" style="border: 5px solid white;"></p> <p>Unfortunately, determining which T cells can target a specific epitope remains challenging, both from a computational and experimental point of view. To address this problem, I developed <strong><a href="https://github.com/GfellerLab/MixTCRpred" rel="external nofollow noopener" target="_blank">MixTCRpred</a>, a state-of-the-art deep learning model that takes the T cell and the epitope sequence as inputs and predicts the likelihood of their interaction.</strong> Accurate prediction can speed up the development of treatments that aim at using or engineering T cells to target specific peptides displayed on MHC molecules, such as cancer neo-epitopes. Have a look at the <a href="https://www.biorxiv.org/content/10.1101/2023.09.13.557561v1" rel="external nofollow noopener" target="_blank">paper</a> and at the code on <a href="https://github.com/GfellerLab/MixTCRpred" rel="external nofollow noopener" target="_blank">Github</a>.</p> <h2 id="modeling-protein-evolution">Modeling protein evolution</h2> <p>Over the course of evolution, genetic mutations can accumulate in the DNA sequences that code for proteins. These mutations can lead to changes in protein amino acid sequences and affect their structure and function. <strong>Is it possible to model protein evolution, i.e. to anticipate which amino acid mutations will appear in the future?</strong></p> <p><img align="center" src="../assets/img/dca_covid.png" width="800" style="border: 5px solid white;"></p> <h4 id="sars-cov-2">SARS-CoV-2</h4> <p>From the beginning of the COVID-19 pandemics, mutations accumulating on SARS-CoV-2 proteins have lead to the emergence of highly infectious and immune-evading SARS-CoV-2 variants, a major concern for public health. We proposed a statistical model trained on sequence data from coronaviruses circulating before the pandemic began to identify protein positions in SARS-CoV-2 proteins are more likely to undergo mutations. Through a retrospective analysis, we proved that our model could accurately predict polymorphic residues that have mutated from the very first known strain of SARS-CoV-2 (Wuhan-Hu-1) during the past years of COVID-19 pandemic, with increased accuracy as more data becomes available. We integrated our predictions with immunological data to pinpoint mutations that are expected to induce immune escape and thus to be over represented in current and future SARS-CoV-2 variants of concern. Here some slides about <a href="https://bevas-epfl.github.io/pdfs/Giancarlo_Croce.pdf" rel="external nofollow noopener" target="_blank">this project</a>. For an introduction to the problem and to our approach, see the articles on <a href="https://www.rfi.fr/en/science-and-technology/20220214-from-the-labs-french-scientists-predict-possible-covid-mutation-sites" rel="external nofollow noopener" target="_blank">RFI</a> (in english) or <a href="https://www.radiofrance.fr/franceculture/podcasts/le-journal-des-sciences/le-journal-des-sciences-du-jeudi-13-janvier-2022-7412422" rel="external nofollow noopener" target="_blank">France Culture</a> (in french).</p> <h4 id="escherichia-coli">Escherichia coli</h4> <p>The idea of using evolutionary sequence data to predict protein evolution can be extended to other cases beyond the SARS-CoV-2 virus. In another project we used the proteomes of 60,000 recently diverged <em>Escherichia coli</em> strains to develop the theoretical framework. See the introductory article on <a href="https://communities.springernature.com/posts/deciphering-polymorphism-in-e-coli" rel="external nofollow noopener" target="_blank">Nature Portfolio</a>.</p> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Giancarlo Croce. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?07b8786bab9b4abe90d10e61f7d12ff7" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-8X320LLJC8"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-8X320LLJC8");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>